apps/cookbook/src/app/app.routes.ts

```diff
@@ -69,6 +70,7 @@ export const routes: Routes = [
   },
   {
     path: 'examples',
-    loadChildren: () => import('./examples/examples.routes').then((m) => m.EXAMPLE_ROUTES),
+    loadChildren: () =>
+      import('./examples/examples-routing.module').then((m) => m.ExamplesRoutingModule),
```

Comment:
Could we lazy load the routes in general, but then make a small subset with the modal routes that we eagerly load, without involving any modules?

I think that woul be the most sleek approach.